Methods to directly add rows and cols to highs model

mmghannam opened the pull request.

lovasoa reviewed (changes requested):
looks good.  Two small things: can we make it very obvious in the doc comments which methods can panic, and under which circumstances ?

Can we add asserts to the tests that check that the right values were added ? And also check the error cases of the try_* functions.

mmghannam commented:
> looks good. Two small things: can we make it very obvious in the doc comments which methods can panic, and under which circumstances ?
> 
I added a panics section to the docs of the methods that can panic. 
The circumstances that they can panic are not in the documentation, I took a look at the highs [code](https://github.com/ERGO-Code/HiGHS/blob/7295b0b37d5b29809aa6d44f215083b2ced182d5/src/lp_data/HighsInterface.cpp#L64); it seems there are many cases but I think some checks are irrelevant here because of the way we call the underlying highs function. 

> Can we add asserts to the tests that check that the right values were added ? And also check the error cases of the try_* functions.
I'm not sure how to do that without implementing `get_cols()` and `get_rows()` methods too. But I added an extra assert for the solution value.

lovasoa commented:
Godd for me, thanks @mmghannam !
